web/js/transcription.js

```diff
@@ -3730,6 +3735,11 @@ $(function() {
             }
             //ESC pressed
             if(event.which === 27) $("#fullscreenBtn").click();
+            if (event.which === 33 || event.which === 34){
```

Comment:
I'm nervous that this needs to be in two places, but I'm willing to not look into it.